zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromIncidentHandlerTest.java

```diff
@@ -25,14 +28,17 @@
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import org.junit.jupiter.api.Test;
 
 public class ListViewProcessInstanceFromIncidentHandlerTest {
 
   private final ProtocolFactory factory = new ProtocolFactory();
   private final String indexName = "test-list-view";
+
+  private final ProcessCache processCache = mock(ProcessCache.class);
```

Comment:
❓ Could we also use a simple fake implementation, based on map? Might make the test easier, as we don't need to mock and write what the mock needs to do, etc. (lots of coupling with the implementation otherwise)